javascript/javascript1/week2/Studentmanager.js

```diff
@@ -0,0 +1,29 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+   if ( studentName >= 6 )
+   return "Cannot add more students to class 07";
+   if (studentName === " ")
+   return "Name is empty. So you can not add to class.";
+   if (studentName === "queen") 
+    return class07Students.push(studentName);
+}
+// expect output 7 names with Queen.
+console.log(class07Students); 
```

Comment:
You were not asked to print the student names, rather the status messages for different cases.